app/controlplane/internal/biz/workflow.go

```diff
@@ -211,6 +212,14 @@ func (uc *WorkflowUseCase) FindByID(ctx context.Context, workflowID string) (*Wo
 	return uc.wfRepo.FindByID(ctx, workflowUUID)
 }
 
+func (uc *WorkflowUseCase) FindByName(ctx context.Context, workflowName string) (*Workflow, error) {
+	if workflowName == "" {
+		return nil, NewErrValidationStr("empty workflow name")
+	}
+
+	return uc.wfRepo.FindByName(ctx, workflowName)
```

Comment:
We need to make sure the workflow we find is in our organization. Note how we have `FindByIDInOrg`